provider/shim/go.mod

```diff
@@ -3,58 +3,73 @@ module github.com/oracle/terraform-provider-oci/shim
 go 1.18
 
 require (
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.14.0
-	github.com/oracle/terraform-provider-oci v1.0.19-0.20220817171459-d3c9b316bc6d
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.22.0
+	github.com/oracle/terraform-provider-oci v1.0.19-0.20220907121719-3199387f2d7f
 )
 
+replace github.com/hashicorp/terraform-exec => github.com/hashicorp/terraform-exec v0.14.0
```

Comment:
This was needed because github.com/hashicorp/terraform-exec has removed `tfinstall` in newer tags and it was required by a dependency.